helm/dex-app/templates/dex-secret.yaml

```diff
@@ -30,13 +31,20 @@ stringData:
       public: true
       name: gscliauth
       secret: {{ .Values.Installation.V1.Secret.OIDC.StaticClients.GSCLIAuth.Secret }}
+    - id: {{ .Values.Installation.V1.GiantSwarm.OIDC.StaticClients.Happa.ClientID }}
+      redirectURIs:
+      - {{ .Values.Installation.V1.GiantSwarm.OIDC.StaticClients.Happa.Address }}
+      - http://localhost:7000/cp-access/callback
```

Comment:
This is for local testing in happa

Reply:
I'm a bit confused about happa's oidc client having a secret. There are different types of auth flows, some that don't require secrets but are still secure, but more complex to implement. 

Having a client with a secret might put is in a flow that we think is more secure, but the secret will be visible in Happa's source code I think.

Reply:
Seems to be the only configuration that `dex` supports. I haven't been able to make it work without a secret in happa, nor in `kubectl-gigantic`

Reply:
ok let's make sure to do a security review at some point, but for now I don't want to block it if that's the way it has to be done for dex